docs/info.md

```diff
@@ -0,0 +1,8 @@
+Grid mission is to empower users with the latest and greatest of the Web 3.0 toolset. 
```

Comment:
Grid's* mission...

I also like the Tron wiki-based quote that Philipp used in his presentation - the bit about "experimental platform where all forms of research could be carried out at unparalleled speeds." maybe from [here?](https://trons.fandom.com/wiki/Grid) Don't necessarily need to copy-paste the quote in, but the idea of it is nice.